### DISCUSSION TITLE
飛機座椅讀取乘客情緒，然後小電一下

### DISCUSSION THREAD

**Discussion by lyien1021** [1101-GS4719-C/Python-Community-GS4719-C#1197] [General]
https://technews.tw/2023/02/22/embraer-airplane-seat-electrify/

巴西航空公司申請一項神奇專利，名叫「提供幸福感的大腦刺激系統」，讓飛機座椅能讀取乘客情緒，焦慮時就提供微電流刺激，使人平靜。

數位相機掃描乘客臉部，臉部辨識工具辨識乘客情緒，介面提供乘客非侵入性經顱刺激，例如直流電刺激、磁刺激和脈衝超音波。

換言之，飛機座椅可「讀取」情緒，知道乘客是否焦慮，並發送電或磁波刺激，讓乘客平靜下來（理論上）。

單看敘述雖然有點嚇人，但經顱磁刺激或微電流刺激是治療憂鬱症療法之一，也是部分人想治療精神健康狀況的選擇。但臉部分析可能涉及隱私和個資安全，所以申請專利後就沒商業化。

----
我覺得雖然這聽起來很酷，但一想到坐飛機的時候臉部還要一直被數位相機拍著，我就覺得蠻焦慮的，這樣會不會一直被電哈哈哈

**Comment by zilinng** [comment]
之前只知道憂鬱症能靠藥物緩解,電流刺激的療法還是第一次知道,讓我長知識了!至於飛機座椅的新功能,雖然聽起來真的好酷,但我覺得座椅會電我這件事本身會讓我更焦慮也><

**Comment by DuxtinTai** [comment]
想到這個畫面就忍不住笑出來。
講認真的，一直被監控真的會讓搭機更焦慮，這項裝置真的有用嗎?還是只是個行銷噱頭?
而且會不會有些個資的問題?
